### ISSUE TITLE
Multiple products

### ISSUE THREAD

**Issue by ravi-rupareliya** [blundell/SimpleInAppPurchaseV3#1]
I have used your library in my project and it is working like charm.
Now problem arise when i am having more than one product for in app purchase, how to deal with that?

**Comment by blundell** [comment]
Make sure you've got the right SKU in use https://github.com/blundell/SimpleInAppPurchaseV3/blob/master/src/com/blundell/tutorial/simpleinappbillingv3/ui/PurchasePassportActivity.java#L39

**Comment by ravi-rupareliya** [comment]
yes i got that but how can we deal with this dynamically when i am having multiple products?

**Comment by blundell** [comment]
I'm not sure without a deep investigation, you need to lookin at using a collection of sku's
